models/event.js

```diff
@@ -1 +1,96 @@
-// needs to register every user that puts an event on their wishlist users: [{ type: Schema.Types.ObjectId, ref: "regUsers"}]
\ No newline at end of file
+// needs to register every user that puts an event on their wishlist users: [{ type: Schema.Types.ObjectId, ref: "regUsers"}]
+import mongoose from "mongoose";
+import moment from "moment";
+
+// Helper variables
+const required = true;
+const unique = true;
+const { Schema } = mongoose;
+const date = moment.utc(); // is this done every time, there's a new instance of Event created?
+// const rootImageUrl = `https://heroku.something.com`; --> we might need this as a pointer for the event images?
+
+// Setting up event schema
+const eventSchema = new Schema({
```

Comment:
indentation for 14,15,16 should be one level less (to the left) :)

Reply:
👍